trace/opencensus/trace/tracer/context_tracer.py

```diff
@@ -78,6 +78,21 @@ def set_enabled(self):
         else:
             return False
 
+    def trace_decorator(self):
+        """Decorator to trace a function."""
+
+        def decorator(func):
+
+            def wrapper(*args, **kwargs):
```

Comment:
Maybe I'm misunderstanding this.  Suppose I have two http request handlers, for `/foo` and `/bar`.  They each call `hello`.  `/foo` has a trace context, `/bar` has a different trace context.  The handlers both call `hello`.  The calls to hello will add spans to a third context, unrelated to the `/foo` and `/bar` contexts?